### DISCUSSION TITLE
Set gcc problem matcher for internal build process

### DISCUSSION THREAD

**Discussion by infn-ke** [microsoft/vscode-cmake-tools#3468] [General]
How can I set gcc problem matcher for internal build process? I'm only able to set problem matcher to gcc when defining a task.json file.

**Comment by gcampbell-msft** [comment]
Is this an issue that you are still facing? I believe this setting allows for what you're asking: `cmake.parseBuildDiagnostics`.

**Reply by infn-ke** [reply]
How does `cmake.parseBuildDiagnostics` interact with UI? Should compiler warnings etc end up under `PROBLEMS` tab? What is default value for this option?

**Reply by gcampbell-msft** [reply]
Yes, the warnings and more should appear in the problems tab. See here: https://github.com/microsoft/vscode-cmake-tools/blob/c848ee3705bcf9580a1ab5c67c5b10c38e02354a/src/cmakeProject.ts#L2062
